pytripgui/canvas_vc/bars/projection_enum.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class BarProjection(Enum):

```

Comment:
What is the reason of adding this enum ?
Shouldn't it be enough to add a plain string attribute to each of `Bar` classes ?

Reply:
There is string attribute in each `Bar` class called `name` that is necessary to register new subclass of `Axes` class. Enum is needed to ensure that name in specific `Bar` class and argument `projection` in `add_axes` method are the same, without any misspelling and to gather all new projection *strategies* in one class

Reply:
![image](https://user-images.githubusercontent.com/44035398/126375401-95084a34-0cbb-4847-84b4-2efe42cf4219.png)